src/queue.ts

```diff
@@ -200,7 +200,7 @@ export class Queue {
     return new Job(payload, commit.hash, nextJobId)
   }
 
-  async markJobAsStarted(payload: string): Promise<CommitInfo> {
+  async markJobAsStarted(id: number, payload: string): Promise<CommitInfo> {
```

Comment:
hi @yeraydavidrodriguez shouldn't be a `JobId` type instead of `number`?

Reply:
Yes, it's fixed